lib/modules/manager/npm/post-update/pnpm.ts

```diff
@@ -68,15 +70,31 @@ export async function generateLockFile(
     }
     const commands: string[] = [];
 
-    cmd = 'pnpm';
-    let args = 'install --recursive --lockfile-only';
+    cmd = 'pnpm install';
+    let args = '--recursive --lockfile-only';
     if (!GlobalConfig.get('allowScripts') || config.ignoreScripts) {
       args += ' --ignore-scripts';
       args += ' --ignore-pnpmfile';
     }
     logger.trace({ cmd, args }, 'pnpm command');
     commands.push(`${cmd} ${args}`);
```

Comment:
I think this should be skipped if only lockfile updates.

Reply:
skipped that command if there are only lockfile updates. Is that what you were thinking about ?

Reply:
yes